### PR TITLE
Python linter [14587]

### PR DESCRIPTION
**Pull request by jparisu**

### CODE REVIEW

**Comment by irenebm** [comment]
clang-tidy failling --- should add `CMakeLists.txt` and `package.xml` to `ampli_py`

File: .github/actions/install-python-packages/action.yml [hunk 1]
```diff
@@ -6,15 +6,19 @@ runs:
...
+          black==22.3.0 \
           colcon-common-extensions \
           colcon-mixin \
-          vcstool \
+          doc8==0.8.0 \
           GitPython \
+          isort==5.10.0 \
+          mypy==0.761 \
+          pylint==2.13.0 \
           setuptools \
```
Comment (irenebm) [inline .github/actions/install-python-packages/action.yml:17]: setuptools==58.2.0 \

File: amlip_py/_setup.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+#     long_description = f.read()
+
+file_packages = [
+    package_name,
+    package_name + '/node',
+    package_name + '/types',
+]
+
+# TODO remove
+variable = 'this variable is veeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeery long for python'
```
Comment (irenebm) [inline amlip_py/_setup.py:18]: should fail?

Comment (jparisu) [reply amlip_py/_setup.py:18]: not anymore. Lenght allowed is up to 99.

**Comment by irenebm** [comment]
LGTM with green CI

**Pull request merged**